tools/build_velox.sh

```diff
@@ -106,8 +106,6 @@ if [ $BUILD_VELOX_FROM_SOURCE == "ON" ]; then
     echo "VELOX_PREFIX=${VELOX_PREFIX}"
     echo "VELOX_SOURCE_DIR=${VELOX_SOURCE_DIR}"
     echo "VELOX_INSTALL_DIR=${VELOX_INSTALL_DIR}"
-    mkdir -p $VELOX_SOURCE_DIR
-    mkdir -p $VELOX_INSTALL_DIR
```

Comment:
VELOX_INSTALL_DIR makes no sense here, do you keep it for further purpose or remove it totally?

Reply:
I am not sure. I can try removing it then see if CI passes.

Reply:
Sorry I recalled the reason why I didn't remove the variable. That it can be reasonable that once we decide to install Velox to some folder rather than using the default install strategy. Does that make sense to you?